include/gsl/gsl_assert

```diff
@@ -68,20 +68,54 @@
 
 namespace gsl
 {
+
 struct fail_fast : public std::runtime_error
 {
     explicit fail_fast(char const* const message) : std::runtime_error(message) {}
 };
-}
+namespace details
+{
+template <typename Exception>
+void throw_exception(Exception&& exception);
```

Comment:
Thanks for the reminder. I never used 2013 significantly, so I forget which features were new in 2015.  I think `[[noreturn]]` for `throw_exception` is significant enough to deserve a `GSL_NORETURN` portability macro until VS2013 support is dropped.